content/en/docs/examples/microservices-istio/setup-kubernetes-cluster/index.md

```diff
@@ -14,8 +14,11 @@ If you are in a workshop and the instructors provide a cluster for you,
 proceed to [setting up your local computer](/docs/examples/microservices-istio/setup-local-computer).
 {{</ warning >}}
 
-1.  Ensure you have access to a [Kubernetes cluster](https://kubernetes.io/docs/tutorials/kubernetes-basics/).
-    You can use the [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart) or the
+1.  Ensure you have access to a
```

Comment:
we already have a platform setup section. Why not just link there?

Reply:
This is specific to this step in the tutorial when using GKE, not a GKE platform prerequisite someone would do in the platform setup document ahead of time.